### PR TITLE
Persons for ba vet120

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**
I have created three names, two of which are brothers according to the text.

### CODE REVIEW

File: new/PRS12978GabraSelassie.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        <revisionDesc>
+            <change who="MKr" when="2020-05-04">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ገብረ፡ ሥላሴ፡</persName>
+                    <persName xml:lang="gez" corresp="#n1">Gabra Śǝllāse</persName>
```
Comment (DariaElagina) [inline new/PRS12978GabraSelassie.xml:49]: this one should contain  type="normalized", and the next empty tag shoud be deleted

File: new/PRS12978GabraSelassie.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
+
+                    <nationality type="Ethiopia"/>
+
+                    <floruit notBefore="1900" notAfter="1940">1900-1940</floruit>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS12978GabraSelassie" passive="BAVet120"/>
+                    <relation name="saws:isRelatedTo" active="PRS12978GabraSelassie" passive="PRS12980WaldaGiyrgis"/>
+                </listRelation>
```
Comment (DariaElagina) [inline new/PRS12978GabraSelassie.xml:59]: https://betamasaheft.eu/Guidelines/?id=relation
the relation saws:isRelatedTo is used for works only. Is the actual relation stated in the manuscript?

File: new/PRS12979GabrielFA.xml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                    <faith type="EOTC"/>
+                    <nationality type="Ethiopia"/>
+                    <birth/>
+                    <death/>
+                    <floruit notBefore="1900" notAfter="1940">1900-1940</floruit>
+                </person>
+                <listRelation>
+                    <relation name="saws:hasOwned" active="PRS12979GabrielFA" passive="BAVet120"/>
+                    <relation name="syriaca:has-relation-to-place" active="PRS12979GabrielFA" passive="LOC3056Farta">
+                    <desc>Referred to as "Gabrǝʾel of Fārṭā Ādiko in Bägemdǝr</desc></relation>
```
Comment (DariaElagina) [inline new/PRS12979GabrielFA.xml:60]: Bagemdǝr, this alternative name can also be added to to the list of persName

File: new/PRS12980WaldaGiyrgis.xml [hunk 4]
```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS12980WaldaGiyrgis" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Wäldä Giyorgis</title>
```
Comment (DariaElagina) [inline new/PRS12980WaldaGiyrgis.xml:8]: Walda

File: new/PRS12980WaldaGiyrgis.xml [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DE" when="2020-05-06">Created entity</change>
+        </revisionDesc>
```
Comment (DariaElagina) [inline new/PRS12980WaldaGiyrgis.xml:42]: MKr

File: new/PRS12980WaldaGiyrgis.xml [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ወልደ፡ ጊዮርጊስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">
+                      Wäldä Giyorgis></persName>
```
Comment (DariaElagina) [inline new/PRS12980WaldaGiyrgis.xml:50]: Walda

File: new/PRS12980WaldaGiyrgis.xml [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+                      <persName xml:lang="gez" xml:id="n2" type="alt">ወርቄ፡ መርሻ፡</persName>
+                      <persName xml:lang="gez" type="normalized" corresp="#n2">
+                        Warqe Meršā></persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
```
Comment (DariaElagina) [inline new/PRS12980WaldaGiyrgis.xml:54]: Maršā

File: new/PRS12980WaldaGiyrgis.xml [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+                    <faith type="EOTC"/>
+                    <nationality type="Ethiopia"/>
+                    <birth/>
+                    <death/>
+                    <floruit notBefore="1900" notAfter="1940">1900-1940</floruit>
+                </person>
+                <listRelation>
+                    <relation name="saws:isRelatedTo" active="PRS12980WaldaGiyrgis" passive="PRS12978GabraSelassie"/>
```
Comment (DariaElagina) [inline new/PRS12980WaldaGiyrgis.xml:62]: If he is mentioned in the ms it is better to encode the relation to the mss, see also a comment above on the saws:isRelatedTo

Comment (MarcinKrawczuk) [reply new/PRS12980WaldaGiyrgis.xml:62]: I wanted to encode that according to the ms. these two are brothers. How do I do that?

File: new/PRS12978GabraSelassie.xml (hunk 2 above)
Comment (MarcinKrawczuk) [reply new/PRS12978GabraSelassie.xml:59]: They were brothers.

File: new/PRS12979GabrielFA.xml (hunk 3 above)
Comment (DariaElagina) [reply new/PRS12979GabrielFA.xml]: I still can`t see the changes, please make sure you did the changes on the right branch

File: new/PRS12980WaldaGiyrgis.xml (hunk 4 above)
Comment (DariaElagina) [reply new/PRS12980WaldaGiyrgis.xml]: same

File: new/PRS12980WaldaGiyrgis.xml (hunk 6 above)
Comment (DariaElagina) [reply new/PRS12980WaldaGiyrgis.xml]: same

**Pull request merged**